Index: .classpath
===================================================================
--- .classpath	(revision 125)
+++ .classpath	(working copy)
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/java-6"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/"/>
 	<classpathentry kind="lib" path="lib/gson-1.7.1-javadoc.jar"/>
 	<classpathentry kind="lib" path="lib/gson-1.7.1-sources.jar"/>
 	<classpathentry kind="lib" path="lib/gson-1.7.1.jar"/>
Index: src/LONI/tree/Connection.java
===================================================================
--- src/LONI/tree/Connection.java	(revision 170)
+++ src/LONI/tree/Connection.java	(working copy)
@@ -1,12 +1,6 @@
-package LONI.tree;
+public class Connection extends LoniNode {
 
 
-public class Connection extends LoniNode {
-	String Source;
-	String Sink;
-	
-	public Connection(String source, String sink){
-		this.Source = source;
-		this.Sink = sink;
-	}
+	private String source;
+	private String sink;
 }
Index: src/LONI/tree/Style.java
===================================================================
--- src/LONI/tree/Style.java	(revision 166)
+++ src/LONI/tree/Style.java	(working copy)
@@ -1,9 +0,0 @@
-package LONI.tree;
-
-public class Style extends LoniNode {
-
-String	Name;
-String	FontSize;
-Boolean	IsBold;
-Boolean	IsItalic;
-}
Index: src/LONI/tree/MetadataActions.java
===================================================================
--- src/LONI/tree/MetadataActions.java	(revision 166)
+++ src/LONI/tree/MetadataActions.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class MetadataActions extends LoniNode {
-
-}
Index: src/LONI/tree/StringExtract.java
===================================================================
--- src/LONI/tree/StringExtract.java	(revision 166)
+++ src/LONI/tree/StringExtract.java	(working copy)
@@ -1,13 +0,0 @@
-package LONI.tree;
-
-public class StringExtract extends LoniNode {
-
-String	Source;
-String	Rule;
-String	StartString;
-String	EndString;
-Boolean	CaseSensitive;
-String	Delimiter;
-Integer	ROW;
-Integer	COL;
-}
Index: src/LONI/tree/ExecutableProvenance.java
===================================================================
--- src/LONI/tree/ExecutableProvenance.java	(revision 166)
+++ src/LONI/tree/ExecutableProvenance.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class ExecutableProvenance extends LoniNode {
-
-}
Index: src/LONI/tree/WebServiceModule.java
===================================================================
--- src/LONI/tree/WebServiceModule.java	(revision 166)
+++ src/LONI/tree/WebServiceModule.java	(working copy)
@@ -1,18 +0,0 @@
-package LONI.tree;
-
-public class WebServiceModule extends LoniNode {
-
-String	Name;
-String	ID;
-String	Description;
-String	Version;
-String	Package;
-String	Icon;
-String	WsdlLocation;
-String	SelectedInterface;
-String	SelectedOperation;
-String	ExecutionServer;
-Integer	Rotation;
-Integer	PosX;
-Integer	PosY;
-}
Index: src/LONI/tree/FileTypes.java
===================================================================
--- src/LONI/tree/FileTypes.java	(revision 166)
+++ src/LONI/tree/FileTypes.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class FileTypes extends LoniNode {
-
-}
Index: src/LONI/tree/Pipeline.java
===================================================================
--- src/LONI/tree/Pipeline.java	(revision 170)
+++ src/LONI/tree/Pipeline.java	(working copy)
@@ -1,6 +1,5 @@
-package LONI.tree;
-
-public class Pipeline extends LoniNode {
-	ModuleGroup workflow;
-	String version;
-}
+public class Pipeline extends LoniNode {
+
+
+	private String version;
+}
Index: src/LONI/tree/ViewerModule.java
===================================================================
--- src/LONI/tree/ViewerModule.java	(revision 170)
+++ src/LONI/tree/ViewerModule.java	(working copy)
@@ -1,15 +1,14 @@
-package LONI.tree;
-
 public class ViewerModule extends LoniNode {
 
-String	ID;
-String	Name;
-String	Package;
-String	Version;
-String	Description;
-String	Location;
-Integer	PosX;
-Integer	PosY;
-Integer	Rotation;
-String	Icon;
+
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String description;
+	private String location;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private String icon;
 }
Index: src/LONI/tree/StudyGroups.java
===================================================================
--- src/LONI/tree/StudyGroups.java	(revision 166)
+++ src/LONI/tree/StudyGroups.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class StudyGroups extends LoniNode {
-
-}
Index: src/LONI/tree/Data.java
===================================================================
--- src/LONI/tree/Data.java	(revision 166)
+++ src/LONI/tree/Data.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Data extends LoniNode {
-
-}
\ No newline at end of file
Index: src/LONI/tree/WsOutputPreference.java
===================================================================
--- src/LONI/tree/WsOutputPreference.java	(revision 166)
+++ src/LONI/tree/WsOutputPreference.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class WsOutputPreference extends LoniNode {
-
-}
Index: src/LONI/tree/Annotation.java
===================================================================
--- src/LONI/tree/Annotation.java	(revision 166)
+++ src/LONI/tree/Annotation.java	(working copy)
@@ -1,14 +0,0 @@
-package LONI.tree;
-
-public class Annotation extends LoniNode {
-
-String	Type;
-Boolean	Expanded;
-String	Name;
-String	Width;
-String	Height;
-String	PosX;
-String	PosY;
-String	Background;
-String	Foreground;
-}
Index: src/LONI/tree/Transform.java
===================================================================
--- src/LONI/tree/Transform.java	(revision 166)
+++ src/LONI/tree/Transform.java	(working copy)
@@ -1,7 +0,0 @@
-package LONI.tree;
-
-public class Transform extends LoniNode {
-
-Integer	Order;
-String	Operation;
-}
Index: src/LONI/tree/MetadataAction.java
===================================================================
--- src/LONI/tree/MetadataAction.java	(revision 166)
+++ src/LONI/tree/MetadataAction.java	(working copy)
@@ -1,7 +0,0 @@
-package LONI.tree;
-
-public class MetadataAction extends LoniNode {
-
-String	Location;
-String	Action;
-}
Index: src/LONI/tree/StudyModule.java
===================================================================
--- src/LONI/tree/StudyModule.java	(revision 170)
+++ src/LONI/tree/StudyModule.java	(working copy)
@@ -1,20 +1,20 @@
-package LONI.tree;
 public class StudyModule extends LoniNode {
 
-String	ID;
-String	Name;
-String	Package;
-String	Version;
-String	Description;
-Integer	PosX;
-Integer	PosY;
-Boolean	IsGroupAsOutput;
-Integer	Rotation;
-String	Type;
-Boolean	DirSource;
-String	ImportStudyRule;
-String	DerivedDataTag;
-Boolean	UseDirSourceFilters;
-Integer	DirSourceFilterType;
-Boolean	Recursive;
-}
\ No newline at end of file
+
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String description;
+	private int posX;
+	private int posY;
+	private boolean isGroupAsOutput;
+	private int rotation;
+	private String type;
+	private boolean dirSource;
+	private String importStudyRule;
+	private String derivedDataTag;
+	private boolean useDirSourceFilters;
+	private int dirSourceFilterType;
+	private boolean recursive;
+}
Index: src/LONI/tree/DataModule.java
===================================================================
--- src/LONI/tree/DataModule.java	(revision 166)
+++ src/LONI/tree/DataModule.java	(working copy)
@@ -1,20 +0,0 @@
-package LONI.tree;
-
-public class DataModule extends LoniNode {
-
-String	ID;
-String	Name;
-String	Package;
-String	Version;
-String	Description;
-Integer	PosX;
-Integer	PosY;
-Integer	Rotation;
-String	Type;
-Boolean	Source;
-Boolean	DirSource;
-Boolean	DirDump;
-Boolean	UseDirSourceFilters;
-Integer	DirSourceFilterType;
-Boolean	Recursive;
-}
\ No newline at end of file
Index: src/LONI/tree/Parameter.java
===================================================================
--- src/LONI/tree/Parameter.java	(revision 166)
+++ src/LONI/tree/Parameter.java	(working copy)
@@ -1,19 +0,0 @@
-package LONI.tree;
-
-public class Parameter extends LoniNode {
-
-String	ID;
-String	Name;
-String	Description;
-Boolean	Enabled;
-Boolean	Required;
-Boolean	Predefined;
-Boolean	IsMetadata;
-Boolean	IsListFile;
-Boolean	IsHideData;
-Boolean	IncludeTransformedParameter;
-Integer	Order;
-String	Prefix;
-Boolean	PrefixSpaced;
-Boolean	PrefixAllArgs;
-}
Index: src/LONI/tree/OS.java
===================================================================
--- src/LONI/tree/OS.java	(revision 166)
+++ src/LONI/tree/OS.java	(working copy)
@@ -1,10 +0,0 @@
-package LONI.tree;
-
-public class OS extends LoniNode {
-
-String	Name;
-String	Version;
-String	Distribution;
-String	KernelName;
-String	KernelVersion;
-}
Index: src/LONI/tree/Connections.java
===================================================================
--- src/LONI/tree/Connections.java	(revision 170)
+++ src/LONI/tree/Connections.java	(working copy)
@@ -1,6 +1,4 @@
-package LONI.tree;
-
-
 public class Connections extends LoniNode {
 
+
 }
Index: src/LONI/tree/Architecture.java
===================================================================
--- src/LONI/tree/Architecture.java	(revision 166)
+++ src/LONI/tree/Architecture.java	(working copy)
@@ -1,9 +0,0 @@
-package LONI.tree;
-
-public class Architecture extends LoniNode {
-
-String	Name;
-String	Vendor;
-String	Model;
-String	ProcessorFlags;
-}
Index: src/LONI/tree/Binary.java
===================================================================
--- src/LONI/tree/Binary.java	(revision 166)
+++ src/LONI/tree/Binary.java	(working copy)
@@ -1,11 +0,0 @@
-package LONI.tree;
-
-public class Binary extends LoniNode {
-
-String	Name;
-String	Description;
-String	Version;
-String	Package;
-String	URI;
-String	CompilationTime;
-}
Index: src/LONI/tree/Authors.java
===================================================================
--- src/LONI/tree/Authors.java	(revision 166)
+++ src/LONI/tree/Authors.java	(working copy)
@@ -1,8 +0,0 @@
-package LONI.tree;
-
-public class Authors extends LoniNode {
-
-String	FullName;
-String	Email;
-String	Website;
-}
Index: src/LONI/tree/Module.java
===================================================================
--- src/LONI/tree/Module.java	(revision 170)
+++ src/LONI/tree/Module.java	(working copy)
@@ -1,18 +1,26 @@
-package LONI.tree;
+public class Module extends LoniNode {
 
-import java.util.List;
 
-public class Module extends LoniNode {
-	String ID;
-	String Location;
-	Integer PositionX;
-	Integer PositionY;
-	public Module(String id, String location, Integer PositionX, Integer PositionY){
-		this.ID = id;
-		this.Location = location;
-		this.PositionX = PositionX;
-		this.PositionY = PositionY;
-	}
-	
-
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String executableVersion;
+	private String description;
+	private String location;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private String icon;
+	private String advancedOptions;
+	private boolean smartModule;
+	private boolean isIDAModule;
+	private boolean isXNATModule;
+	private boolean isRepeatUntil;
+	private String sourceCode;
+	private boolean requireNetAccess;
+	private boolean MPIEnabled;
+	private String MPIParallelEnv;
+	private String MPINumSlots;
+	private boolean preserveInputFilename;
 }
Index: src/LONI/tree/Variable.java
===================================================================
--- src/LONI/tree/Variable.java	(revision 166)
+++ src/LONI/tree/Variable.java	(working copy)
@@ -1,10 +0,0 @@
-package LONI.tree;
-
-public class Variable extends LoniNode {
-
-String	Name;
-String	Description;
-Boolean	Required;
-Boolean	Secret;
-Integer	Order;
-}
Index: src/LONI/tree/LoopGroup.java
===================================================================
--- src/LONI/tree/LoopGroup.java	(revision 170)
+++ src/LONI/tree/LoopGroup.java	(working copy)
@@ -1,14 +1,14 @@
-package LONI.tree;
+public class LoopGroup extends LoniNode {
 
 
-public class LoopGroup extends LoniNode{
-	String Name;
-	String ID;
-	String Package;
-	String Version;
-	String Description;
-	Integer PositionX;
-	Integer PositionY;
-	Boolean NeedsProvenance;
-	//List<Loop> Loops;	
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String description;
+	private String icon;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private boolean needsProvenance;
 }
Index: src/LONI/tree/Dependencies.java
===================================================================
--- src/LONI/tree/Dependencies.java	(revision 166)
+++ src/LONI/tree/Dependencies.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Dependencies extends LoniNode {
-
-}
Index: src/LONI/tree/WsOutputPreferences.java
===================================================================
--- src/LONI/tree/WsOutputPreferences.java	(revision 166)
+++ src/LONI/tree/WsOutputPreferences.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class WsOutputPreferences extends LoniNode {
-
-}
Index: src/LONI/tree/Citations.java
===================================================================
--- src/LONI/tree/Citations.java	(revision 166)
+++ src/LONI/tree/Citations.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Citations extends LoniNode {
-
-}
Index: src/LONI/tree/Values.java
===================================================================
--- src/LONI/tree/Values.java	(revision 166)
+++ src/LONI/tree/Values.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Values extends LoniNode {
-
-}
Index: src/LONI/tree/ModuleGroup.java
===================================================================
--- src/LONI/tree/ModuleGroup.java	(revision 170)
+++ src/LONI/tree/ModuleGroup.java	(working copy)
@@ -1,22 +1,14 @@
-package LONI.tree;
+public class ModuleGroup extends LoniNode {
 
-import java.util.LinkedList;
-import java.util.List;
 
-public class ModuleGroup extends LoniNode{
-	String Name;
-	String ID;
-	String Package;
-	String Version;
-	String Description;
-	Integer PositionX;
-	Integer PositionY;
-	List<Module> Modules;	// How do judge to creat this? because it's group?
-	public ModuleGroup(){
-		Modules = new LinkedList();
-	}
-	public void addModule(Module m){
-		Modules.add(m);
-	}
-
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String description;
+	private String icon;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private boolean needsProvenance;
 }
Index: src/LONI/tree/Library.java
===================================================================
--- src/LONI/tree/Library.java	(revision 166)
+++ src/LONI/tree/Library.java	(working copy)
@@ -1,10 +0,0 @@
-package LONI.tree;
-
-public class Library extends LoniNode {
-
-String	Name;
-String	Version;
-String	Package;
-String	URI;
-String	CompilationTime;
-}
Index: src/LONI/tree/DirSourceFilter.java
===================================================================
--- src/LONI/tree/DirSourceFilter.java	(revision 166)
+++ src/LONI/tree/DirSourceFilter.java	(working copy)
@@ -1,6 +0,0 @@
-package LONI.tree;
-
-public class DirSourceFilter extends LoniNode {
-
-String	Condition;
-}
Index: src/LONI/tree/LoniNode.java
===================================================================
--- src/LONI/tree/LoniNode.java	(revision 170)
+++ src/LONI/tree/LoniNode.java	(working copy)
@@ -1,7 +1,4 @@
-package LONI.tree;
 
-import Core.Node;
+public class LoniNode extends Node {
 
-public class LoniNode extends Node{
-
 }
Index: src/LONI/tree/Variables.java
===================================================================
--- src/LONI/tree/Variables.java	(revision 166)
+++ src/LONI/tree/Variables.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Variables extends LoniNode {
-
-}
Index: src/LONI/tree/ConditionalModule.java
===================================================================
--- src/LONI/tree/ConditionalModule.java	(revision 170)
+++ src/LONI/tree/ConditionalModule.java	(working copy)
@@ -1,18 +1,17 @@
-package LONI.tree;
-
 public class ConditionalModule extends LoniNode {
 
-String	ID;
-String	Name;
-String	Package;
-String	Version;
-String	ExecutableVersion;
-String	Description;
-String	Location;
-Integer	PosX;
-Integer	PosY;
-Integer	Rotation;
-String	Icon;
-String	AdvancedOptions;
-Boolean	HasMetadata;
+
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String executableVersion;
+	private String description;
+	private String location;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private String icon;
+	private String advancedOptions;
+	private boolean hasMetadata;
 }
Index: src/LONI/tree/SystemConfig.java
===================================================================
--- src/LONI/tree/SystemConfig.java	(revision 166)
+++ src/LONI/tree/SystemConfig.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class SystemConfig extends LoniNode {
-
-}
Index: src/LONI/tree/Compiler.java
===================================================================
--- src/LONI/tree/Compiler.java	(revision 166)
+++ src/LONI/tree/Compiler.java	(working copy)
@@ -1,9 +0,0 @@
-package LONI.tree;
-
-public class Compiler extends LoniNode {
-
-String	Name;
-String	Version;
-String	Flags;
-String	Updates;
-}
Index: src/LONI/tree/Value.java
===================================================================
--- src/LONI/tree/Value.java	(revision 166)
+++ src/LONI/tree/Value.java	(working copy)
@@ -1,6 +0,0 @@
-package LONI.tree;
-
-public class Value extends LoniNode {
-
-String	Metadata;
-}
Index: src/LONI/tree/MetadataValue.java
===================================================================
--- src/LONI/tree/MetadataValue.java	(revision 166)
+++ src/LONI/tree/MetadataValue.java	(working copy)
@@ -1,7 +0,0 @@
-package LONI.tree;
-
-public class MetadataValue extends LoniNode {
-
-String	ValueOption;
-String	ValueValue;
-}
Index: src/LONI/tree/Metadata.java
===================================================================
--- src/LONI/tree/Metadata.java	(revision 166)
+++ src/LONI/tree/Metadata.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Metadata extends LoniNode {
-
-}
Index: src/LONI/tree/FileType.java
===================================================================
--- src/LONI/tree/FileType.java	(revision 166)
+++ src/LONI/tree/FileType.java	(working copy)
@@ -1,8 +0,0 @@
-package LONI.tree;
-
-public class FileType extends LoniNode {
-
-String	Name;
-String	Extension;
-String	Description;
-}
Index: src/LONI/tree/MetadataName.java
===================================================================
--- src/LONI/tree/MetadataName.java	(revision 166)
+++ src/LONI/tree/MetadataName.java	(working copy)
@@ -1,7 +0,0 @@
-package LONI.tree;
-
-public class MetadataName extends LoniNode {
-
-String	NameOption;
-String	NameValue;
-}
Index: src/LONI/tree/Styles.java
===================================================================
--- src/LONI/tree/Styles.java	(revision 166)
+++ src/LONI/tree/Styles.java	(working copy)
@@ -1,5 +0,0 @@
-package LONI.tree;
-
-public class Styles extends LoniNode {
-
-}
Index: src/LONI/tree/StudyGroup.java
===================================================================
--- src/LONI/tree/StudyGroup.java	(revision 166)
+++ src/LONI/tree/StudyGroup.java	(working copy)
@@ -1,7 +0,0 @@
-package LONI.tree;
-
-public class StudyGroup extends LoniNode {
-
-String	Name;
-String	Query;
-}
Index: src/LONI/tree/Format.java
===================================================================
--- src/LONI/tree/Format.java	(revision 166)
+++ src/LONI/tree/Format.java	(working copy)
@@ -1,9 +0,0 @@
-package LONI.tree;
-
-public class Format extends LoniNode {
-
-String	Type;
-Integer	Cardinality;
-String	CardinalityBase;
-String	TransformationBase;
-}
Index: src/LONI/tree/MetadataElement.java
===================================================================
--- src/LONI/tree/MetadataElement.java	(revision 166)
+++ src/LONI/tree/MetadataElement.java	(working copy)
@@ -1,8 +0,0 @@
-package LONI.tree;
-
-public class MetadataElement extends LoniNode {
-
-String	Location;
-String	Prefix;
-Boolean	SpaceAfterPrefix;
-}
Index: src/LONI/tree/Annotations.java
===================================================================
--- src/LONI/tree/Annotations.java	(revision 166)
+++ src/LONI/tree/Annotations.java	(working copy)
@@ -1,6 +0,0 @@
-package LONI.tree;
-
-public class Annotations extends LoniNode{
-	
-	Boolean	Visible;
-}
Index: src/LONI/unNeeded Tree/Style.java
===================================================================
--- src/LONI/unNeeded Tree/Style.java	(revision 0)
+++ src/LONI/unNeeded Tree/Style.java	(working copy)
@@ -1,9 +1,8 @@
-package LONI.tree;
-
 public class Style extends LoniNode {
 
-String	Name;
-String	FontSize;
-Boolean	IsBold;
-Boolean	IsItalic;
+
+	private String name;
+	private String fontSize;
+	private boolean isBold;
+	private boolean isItalic;
 }
Index: src/LONI/unNeeded Tree/MetadataActions.java
===================================================================
--- src/LONI/unNeeded Tree/MetadataActions.java	(revision 0)
+++ src/LONI/unNeeded Tree/MetadataActions.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class MetadataActions extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/StringExtract.java
===================================================================
--- src/LONI/unNeeded Tree/StringExtract.java	(revision 0)
+++ src/LONI/unNeeded Tree/StringExtract.java	(working copy)
@@ -1,13 +1,12 @@
-package LONI.tree;
-
 public class StringExtract extends LoniNode {
 
-String	Source;
-String	Rule;
-String	StartString;
-String	EndString;
-Boolean	CaseSensitive;
-String	Delimiter;
-Integer	ROW;
-Integer	COL;
+
+	private String source;
+	private String rule;
+	private String startString;
+	private String endString;
+	private boolean caseSensitive;
+	private String delimiter;
+	private int row;
+	private int col;
 }
Index: src/LONI/unNeeded Tree/ExecutableProvenance.java
===================================================================
--- src/LONI/unNeeded Tree/ExecutableProvenance.java	(revision 0)
+++ src/LONI/unNeeded Tree/ExecutableProvenance.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class ExecutableProvenance extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/WebServiceModule.java
===================================================================
--- src/LONI/unNeeded Tree/WebServiceModule.java	(revision 0)
+++ src/LONI/unNeeded Tree/WebServiceModule.java	(working copy)
@@ -1,18 +1,17 @@
-package LONI.tree;
-
 public class WebServiceModule extends LoniNode {
 
-String	Name;
-String	ID;
-String	Description;
-String	Version;
-String	Package;
-String	Icon;
-String	WsdlLocation;
-String	SelectedInterface;
-String	SelectedOperation;
-String	ExecutionServer;
-Integer	Rotation;
-Integer	PosX;
-Integer	PosY;
+
+	private String name;
+	private String id;
+	private String description;
+	private String version;
+	private String package;
+	private String icon;
+	private String wsdlLocation;
+	private String selectedInterface;
+	private String selectedOperation;
+	private String executionServer;
+	private int rotation;
+	private int posX;
+	private int posY;
 }
Index: src/LONI/unNeeded Tree/FileTypes.java
===================================================================
--- src/LONI/unNeeded Tree/FileTypes.java	(revision 0)
+++ src/LONI/unNeeded Tree/FileTypes.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class FileTypes extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/StudyGroups.java
===================================================================
--- src/LONI/unNeeded Tree/StudyGroups.java	(revision 0)
+++ src/LONI/unNeeded Tree/StudyGroups.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class StudyGroups extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/WsOutputPreference.java
===================================================================
--- src/LONI/unNeeded Tree/WsOutputPreference.java	(revision 0)
+++ src/LONI/unNeeded Tree/WsOutputPreference.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class WsOutputPreference extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Data.java
===================================================================
--- src/LONI/unNeeded Tree/Data.java	(revision 0)
+++ src/LONI/unNeeded Tree/Data.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Data extends LoniNode {
 
-}
\ No newline at end of file
+
+}
Index: src/LONI/unNeeded Tree/Annotation.java
===================================================================
--- src/LONI/unNeeded Tree/Annotation.java	(revision 0)
+++ src/LONI/unNeeded Tree/Annotation.java	(working copy)
@@ -1,14 +1,13 @@
-package LONI.tree;
-
 public class Annotation extends LoniNode {
 
-String	Type;
-Boolean	Expanded;
-String	Name;
-String	Width;
-String	Height;
-String	PosX;
-String	PosY;
-String	Background;
-String	Foreground;
+
+	private String type;
+	private boolean expanded;
+	private String name;
+	private String width;
+	private String height;
+	private String posX;
+	private String posY;
+	private String background;
+	private String foreground;
 }
Index: src/LONI/unNeeded Tree/Transform.java
===================================================================
--- src/LONI/unNeeded Tree/Transform.java	(revision 0)
+++ src/LONI/unNeeded Tree/Transform.java	(working copy)
@@ -1,7 +1,6 @@
-package LONI.tree;
-
 public class Transform extends LoniNode {
 
-Integer	Order;
-String	Operation;
+
+	private int order;
+	private String operation;
 }
Index: src/LONI/unNeeded Tree/MetadataAction.java
===================================================================
--- src/LONI/unNeeded Tree/MetadataAction.java	(revision 0)
+++ src/LONI/unNeeded Tree/MetadataAction.java	(working copy)
@@ -1,7 +1,6 @@
-package LONI.tree;
-
 public class MetadataAction extends LoniNode {
 
-String	Location;
-String	Action;
+
+	private String location;
+	private String action;
 }
Index: src/LONI/unNeeded Tree/DataModule.java
===================================================================
--- src/LONI/unNeeded Tree/DataModule.java	(revision 0)
+++ src/LONI/unNeeded Tree/DataModule.java	(working copy)
@@ -1,20 +1,19 @@
-package LONI.tree;
-
 public class DataModule extends LoniNode {
 
-String	ID;
-String	Name;
-String	Package;
-String	Version;
-String	Description;
-Integer	PosX;
-Integer	PosY;
-Integer	Rotation;
-String	Type;
-Boolean	Source;
-Boolean	DirSource;
-Boolean	DirDump;
-Boolean	UseDirSourceFilters;
-Integer	DirSourceFilterType;
-Boolean	Recursive;
-}
\ No newline at end of file
+
+	private String id;
+	private String name;
+	private String package;
+	private String version;
+	private String description;
+	private int posX;
+	private int posY;
+	private int rotation;
+	private String type;
+	private boolean source;
+	private boolean dirSource;
+	private boolean dirDump;
+	private boolean useDirSourceFilters;
+	private int dirSourceFilterType;
+	private boolean recursive;
+}
Index: src/LONI/unNeeded Tree/Parameter.java
===================================================================
--- src/LONI/unNeeded Tree/Parameter.java	(revision 0)
+++ src/LONI/unNeeded Tree/Parameter.java	(working copy)
@@ -1,19 +1,18 @@
-package LONI.tree;
-
 public class Parameter extends LoniNode {
 
-String	ID;
-String	Name;
-String	Description;
-Boolean	Enabled;
-Boolean	Required;
-Boolean	Predefined;
-Boolean	IsMetadata;
-Boolean	IsListFile;
-Boolean	IsHideData;
-Boolean	IncludeTransformedParameter;
-Integer	Order;
-String	Prefix;
-Boolean	PrefixSpaced;
-Boolean	PrefixAllArgs;
+
+	private String id;
+	private String name;
+	private String description;
+	private boolean enabled;
+	private boolean required;
+	private boolean predefined;
+	private boolean isMetadata;
+	private boolean isListFile;
+	private boolean isHideData;
+	private boolean includeTransformedParameter;
+	private int order;
+	private String prefix;
+	private boolean prefixSpaced;
+	private boolean prefixAllArgs;
 }
Index: src/LONI/unNeeded Tree/OS.java
===================================================================
--- src/LONI/unNeeded Tree/OS.java	(revision 0)
+++ src/LONI/unNeeded Tree/OS.java	(working copy)
@@ -1,10 +1,9 @@
-package LONI.tree;
-
 public class OS extends LoniNode {
 
-String	Name;
-String	Version;
-String	Distribution;
-String	KernelName;
-String	KernelVersion;
+
+	private String name;
+	private String version;
+	private String distribution;
+	private String kernelName;
+	private String kernelVersion;
 }
Index: src/LONI/unNeeded Tree/Architecture.java
===================================================================
--- src/LONI/unNeeded Tree/Architecture.java	(revision 0)
+++ src/LONI/unNeeded Tree/Architecture.java	(working copy)
@@ -1,9 +1,8 @@
-package LONI.tree;
-
 public class Architecture extends LoniNode {
 
-String	Name;
-String	Vendor;
-String	Model;
-String	ProcessorFlags;
+
+	private String name;
+	private String vendor;
+	private String model;
+	private String processorFlags;
 }
Index: src/LONI/unNeeded Tree/Binary.java
===================================================================
--- src/LONI/unNeeded Tree/Binary.java	(revision 0)
+++ src/LONI/unNeeded Tree/Binary.java	(working copy)
@@ -1,11 +1,10 @@
-package LONI.tree;
-
 public class Binary extends LoniNode {
 
-String	Name;
-String	Description;
-String	Version;
-String	Package;
-String	URI;
-String	CompilationTime;
+
+	private String name;
+	private String description;
+	private String version;
+	private String package;
+	private String uri;
+	private String compilationTime;
 }
Index: src/LONI/unNeeded Tree/Variable.java
===================================================================
--- src/LONI/unNeeded Tree/Variable.java	(revision 0)
+++ src/LONI/unNeeded Tree/Variable.java	(working copy)
@@ -1,10 +1,9 @@
-package LONI.tree;
-
 public class Variable extends LoniNode {
 
-String	Name;
-String	Description;
-Boolean	Required;
-Boolean	Secret;
-Integer	Order;
+
+	private String name;
+	private String description;
+	private boolean required;
+	private boolean secret;
+	private int order;
 }
Index: src/LONI/unNeeded Tree/Authors.java
===================================================================
--- src/LONI/unNeeded Tree/Authors.java	(revision 0)
+++ src/LONI/unNeeded Tree/Authors.java	(working copy)
@@ -1,8 +1,7 @@
-package LONI.tree;
-
 public class Authors extends LoniNode {
 
-String	FullName;
-String	Email;
-String	Website;
+
+	private String fullName;
+	private String email;
+	private String website;
 }
Index: src/LONI/unNeeded Tree/Dependencies.java
===================================================================
--- src/LONI/unNeeded Tree/Dependencies.java	(revision 0)
+++ src/LONI/unNeeded Tree/Dependencies.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Dependencies extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/WsOutputPreferences.java
===================================================================
--- src/LONI/unNeeded Tree/WsOutputPreferences.java	(revision 0)
+++ src/LONI/unNeeded Tree/WsOutputPreferences.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class WsOutputPreferences extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Citations.java
===================================================================
--- src/LONI/unNeeded Tree/Citations.java	(revision 0)
+++ src/LONI/unNeeded Tree/Citations.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Citations extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Values.java
===================================================================
--- src/LONI/unNeeded Tree/Values.java	(revision 0)
+++ src/LONI/unNeeded Tree/Values.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Values extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Library.java
===================================================================
--- src/LONI/unNeeded Tree/Library.java	(revision 0)
+++ src/LONI/unNeeded Tree/Library.java	(working copy)
@@ -1,10 +1,8 @@
-package LONI.tree;
-
 public class Library extends LoniNode {
 
-String	Name;
-String	Version;
-String	Package;
-String	URI;
-String	CompilationTime;
-}
+
+	private String name;
+	private String version;
+	private String package;
+	private String uri;
+	private String compilationTime;
Index: src/LONI/unNeeded Tree/DirSourceFilter.java
===================================================================
--- src/LONI/unNeeded Tree/DirSourceFilter.java	(revision 0)
+++ src/LONI/unNeeded Tree/DirSourceFilter.java	(working copy)
@@ -1,6 +1,5 @@
-package LONI.tree;
-
 public class DirSourceFilter extends LoniNode {
 
-String	Condition;
+
+	private String condition;
 }
Index: src/LONI/unNeeded Tree/Variables.java
===================================================================
--- src/LONI/unNeeded Tree/Variables.java	(revision 0)
+++ src/LONI/unNeeded Tree/Variables.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Variables extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Compiler.java
===================================================================
--- src/LONI/unNeeded Tree/Compiler.java	(revision 0)
+++ src/LONI/unNeeded Tree/Compiler.java	(working copy)
@@ -1,9 +1,8 @@
-package LONI.tree;
-
 public class Compiler extends LoniNode {
 
-String	Name;
-String	Version;
-String	Flags;
-String	Updates;
+
+	private String name;
+	private String version;
+	private String flags;
+	private String updates;
 }
Index: src/LONI/unNeeded Tree/SystemConfig.java
===================================================================
--- src/LONI/unNeeded Tree/SystemConfig.java	(revision 0)
+++ src/LONI/unNeeded Tree/SystemConfig.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class SystemConfig extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Value.java
===================================================================
--- src/LONI/unNeeded Tree/Value.java	(revision 0)
+++ src/LONI/unNeeded Tree/Value.java	(working copy)
@@ -1,6 +1,5 @@
-package LONI.tree;
-
 public class Value extends LoniNode {
 
-String	Metadata;
+
+	private String metadata;
 }
Index: src/LONI/unNeeded Tree/MetadataValue.java
===================================================================
--- src/LONI/unNeeded Tree/MetadataValue.java	(revision 0)
+++ src/LONI/unNeeded Tree/MetadataValue.java	(working copy)
@@ -1,7 +1,6 @@
-package LONI.tree;
-
 public class MetadataValue extends LoniNode {
 
-String	ValueOption;
-String	ValueValue;
+
+	private String valueOption;
+	private String valueValue;
 }
Index: src/LONI/unNeeded Tree/Metadata.java
===================================================================
--- src/LONI/unNeeded Tree/Metadata.java	(revision 0)
+++ src/LONI/unNeeded Tree/Metadata.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Metadata extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/FileType.java
===================================================================
--- src/LONI/unNeeded Tree/FileType.java	(revision 0)
+++ src/LONI/unNeeded Tree/FileType.java	(working copy)
@@ -1,8 +1,7 @@
-package LONI.tree;
-
 public class FileType extends LoniNode {
 
-String	Name;
-String	Extension;
-String	Description;
+
+	private String name;
+	private String extension;
+	private String description;
 }
Index: src/LONI/unNeeded Tree/MetadataName.java
===================================================================
--- src/LONI/unNeeded Tree/MetadataName.java	(revision 0)
+++ src/LONI/unNeeded Tree/MetadataName.java	(working copy)
@@ -1,7 +1,6 @@
-package LONI.tree;
-
 public class MetadataName extends LoniNode {
 
-String	NameOption;
-String	NameValue;
+
+	private String nameOption;
+	private String nameValue;
 }
Index: src/LONI/unNeeded Tree/Styles.java
===================================================================
--- src/LONI/unNeeded Tree/Styles.java	(revision 0)
+++ src/LONI/unNeeded Tree/Styles.java	(working copy)
@@ -1,5 +1,4 @@
-package LONI.tree;
-
 public class Styles extends LoniNode {
 
+
 }
Index: src/LONI/unNeeded Tree/Format.java
===================================================================
--- src/LONI/unNeeded Tree/Format.java	(revision 0)
+++ src/LONI/unNeeded Tree/Format.java	(working copy)
@@ -1,9 +1,8 @@
-package LONI.tree;
-
 public class Format extends LoniNode {
 
-String	Type;
-Integer	Cardinality;
-String	CardinalityBase;
-String	TransformationBase;
+
+	private String type;
+	private int cardinality;
+	private String cardinalityBase;
+	private String transformationBase;
 }
Index: src/LONI/unNeeded Tree/MetadataElement.java
===================================================================
--- src/LONI/unNeeded Tree/MetadataElement.java	(revision 0)
+++ src/LONI/unNeeded Tree/MetadataElement.java	(working copy)
@@ -1,8 +1,7 @@
-package LONI.tree;
-
 public class MetadataElement extends LoniNode {
 
-String	Location;
-String	Prefix;
-Boolean	SpaceAfterPrefix;
+
+	private String location;
+	private String prefix;
+	private boolean spaceAfterPrefix;
 }
Index: src/LONI/unNeeded Tree/Annotations.java
===================================================================
--- src/LONI/unNeeded Tree/Annotations.java	(revision 0)
+++ src/LONI/unNeeded Tree/Annotations.java	(working copy)
@@ -1,6 +1,5 @@
-package LONI.tree;
+public class Annotations extends LoniNode {
 
-public class Annotations extends LoniNode{
-	
-	Boolean	Visible;
+
+	private boolean visible;
 }
Index: .settings/org.eclipse.jdt.core.prefs
===================================================================
--- .settings/org.eclipse.jdt.core.prefs	(revision 166)
+++ .settings/org.eclipse.jdt.core.prefs	(working copy)
@@ -1,12 +1,12 @@
-#Mon Oct 24 11:09:00 PDT 2011
-eclipse.preferences.version=1
-org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
-org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
-org.eclipse.jdt.core.compiler.compliance=1.6
-org.eclipse.jdt.core.compiler.debug.lineNumber=generate
-org.eclipse.jdt.core.compiler.debug.localVariable=generate
-org.eclipse.jdt.core.compiler.debug.sourceFile=generate
-org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
-org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.source=1.6
+#Tue Oct 25 17:03:52 PDT 2011
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.5
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
+org.eclipse.jdt.core.compiler.compliance=1.5
+org.eclipse.jdt.core.compiler.debug.lineNumber=generate
+org.eclipse.jdt.core.compiler.debug.localVariable=generate
+org.eclipse.jdt.core.compiler.debug.sourceFile=generate
+org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+org.eclipse.jdt.core.compiler.source=1.5
